src/common_runtime/interface/event_looper.h

```diff
@@ -0,0 +1,42 @@
+# pragma once 
+
+/// error code
+#define CR_OK 0
```

Comment:
Let's avoid abbreviations unless they are very well known. So throughout, `cr` should be `common_runtime`.